SlicerCART/documentation/installation.md

```diff
@@ -0,0 +1,71 @@
+# SlicerCART Installation
+
+## Requirements
+
+* MacOS Sonoma or Sequoia (15.0.1) is recommended 
+* A working version of [3D Slicer](https://download.slicer.org) (version 5.6.2).
+* Qt: might need to be installed.
+  * The first five steps of the following procedure might be useful for this: [procedure](https://web.stanford.edu/dept/cs_edu/resources/qt/install-mac).
+  * You can try importing Qt in the Slicer python console (e.g. `import slicer ` then `from slicer.util import Qt`)
+
+*Although it may work on other versions and/or operating system, please note 
+that it has not been tested.
+
+If 3D Slicer has not been already installed, you can follow these steps: 
+1. Install [3D Slicer](https://download.slicer.org):
+2. Make sure that you are able to open and use the 3D Slicer software before 
+   trying installing any extension/module. 
+3. If you encounter some problems, you are encouraged to refer to:
+   * [3D Slicer Documentation](https://slicer.readthedocs.io/en/latest/)
+   * [3D Slicer forum](https://discourse.slicer.org/) (very active community)
+
+## Installation Steps
+
+1. Clone the [SlicerCART repository](https://github.com/neuropoly/slicer-manual-annotation) in the location of your choice.
+2. Open 3D Slicer. 
+3. Activate the checkbox `Enable developer mode` in `Edit 
+   -> Application 
+    Settings -> Developer -> Enable developer mode`. 
+4. Open Finder (on macOS). Go to the location of the SlicerCART Repository. 
+   Go to the location of the python file `SlicerCART.py` (_should only have one 
+   FILE of that name. Note that if you add the path of the folder SlicerCART,
+   it will not work: you MUST add the path of SlicerCART.py FILE_)  in `Edit -> Application Settings 
+   -> Modules -> Additional module paths`. See image example below:
+![](images/module_filepath.png)
```

Comment:
ok! taken in notes. Will adapt all screenshot in a further commit. will replace by a and b. I would keep the text however since some might only look at the figures!